### PR TITLE
Phpstan: enabled bleeding edge rules

### PR DESCRIPTION
**Pull request by adaamz**

### CODE REVIEW

File: phpstan.neon.dist [hunk 1]
```diff
@@ -1,4 +1,5 @@
 includes:
+	- phar://%rootDir%/phpstan.phar/conf/bleedingEdge.neon
```
Comment (hrach) [inline phpstan.neon.dist]: Seems this does not work

File: phpstan.neon.dist [hunk 2]
```diff
@@ -9,6 +10,13 @@ parameters:
 	ignoreErrors:
 		# https://github.com/phpstan/phpstan/issues/587
 		- '#Constructor of class Nextras\\Orm\\Bridges\\NetteDI\\DIRepositoryFinder has an unused parameter \$modelClass\.#'
+		-
+			message: '#^Instanceof between Nette\\DI\\Definitions\\Definition and Nette\\DI\\Definitions\\ServiceDefinition will always evaluate to false\.$#'
```
Comment (hrach) [inline phpstan.neon.dist]: Please add comment ``# Compatibility fix with Nette 2.4`

File: phpstan.neon.dist (hunk 1 above)
Comment (adaamz) [reply phpstan.neon.dist]: yes, bleeding edge rules are there from 0.11.9...
i will wait for https://github.com/nextras/orm/pull/355 and then rebase this pr

Comment (hrach) [reply phpstan.neon.dist]: Thanks, done.

**Pull request merged**